README.md

```diff
@@ -3,22 +3,17 @@
 
 ## Intro
 
-PnP is a set of technologies designed to make it easier to build IoT Solutions based on Azure IoT Hub. To address the needs for device and cloud developers the tutorial must target both.
-
-Devices connected to IoT Hub can de described with DTDL, in terms of:
-
-- Properties (Available trough Twins)
-- Commands (Using the Hub Direct Methods feature)
-- Telemetry (processed through EventHub)
+This bug bash explores the new IoT Hub features to support the Plug and Play May 2020 release. Using a  DTDL v2 Model to describe a device with one  sensor interface that sends telemetry, accepts commands and reports properties. The sample is based on a device simulator connected to IoT Hub with a client SDK and can be managed using the service SDKs  to read telemetry and interact with properties and commands.
 
 Review the docs in the private staging environment [IoT Plug and Play Preview Documentation](https://review.docs.microsoft.com/azure/iot-pnp/?branch=pr-en-us-114283)
 
+[DTDL v2 Spec](https://microsoft.sharepoint.com/:w:/t/Azure_IoT/IoTPlat/EaotICYXiQVOh_xnBggJwqMBMKGoOyMzV1qFYsOdcyvOVw?e=VyG0QN) (internal)
+
 ### BugBash support and feedback
 
-- The teams channel [PnP Public Preview BugBash](https://teams.microsoft.com/l/channel/19%3a0b9d0f166a3d41c69ce90fcca7631962%40thread.skype/PnP%2520Public%2520Preview?groupId=dcc1ac84-f476-4c96-8034-b2d77e54c8bf&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47). Will have team members answering questions during the BugBash.
+- Use the teams channel [PnP Public Preview BugBash](https://teams.microsoft.com/l/channel/19%3a0b9d0f166a3d41c69ce90fcca7631962%40thread.skype/PnP%2520Public%2520Preview?groupId=dcc1ac84-f476-4c96-8034-b2d77e54c8bf&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) to ask for help. There will be team members answering questions during the BugBash.
 - If you have bugs or feature request, please use this [Bug Template](https://msazure.visualstudio.com/One/_workitems/create/Bug?templateId=a05dbe62-eb63-4028-9ce0-252c552872f3&ownerId=1e65a829-00c0-4dc9-8088-d41678a0d033). You can query existing bugs in this [PnP BugBash Query](https://msazure.visualstudio.com/One/_queries/query-edit/71cca2fe-63bd-4916-84f6-937f0dfaa698/?action=new)
```

Comment:
The bug template is pointing to iteration for April 2020 not May.

Reply:
@miagdp can you update the bug template and query?

Reply:
@miagdp can you update the bug template?

Reply:
Fixed.